test/typings/rison_node.d.ts

```diff
@@ -18,7 +18,7 @@
  */
 
 declare module 'rison-node' {
-  export type RisonValue = null | boolean | number | string | RisonObject | RisonArray;
+  export type RisonValue = undefined | null | boolean | number | string | RisonObject | RisonArray;
```

Comment:
rison-whack-a-mole ... `undefined` is totally valid for Rison values, whoever setup this up just never had a type with option values being set.

Reply:
OK, I feel like we've edited various versions of this file in various places in Kibana at least 4-5 times since I started, let's just keep whacking this mole for now! 🔨 🔨 🔨